tarsier-service/service/src/db.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Any
+from sqlalchemy import create_engine, text, Row
+import uuid
+import json
+
+host = os.environ.get("DB_HOST") or "localhost"
+engine = create_engine(f"mysql+pymysql://user:password@{host}:3306/db") #docker compose ...@db:3306/db"
+
+with engine.connect() as conn:
+    # conn.execute(text('DROP TABLE jobs'))
+    conn.execute(text('''
+        CREATE TABLE IF NOT EXISTS jobs (
+            id VARCHAR(255) PRIMARY KEY,
+            user_id VARCHAR(60),
+            url VARCHAR(2083),
+            page_text TEXT,
+            outputSchema JSON,
+            output JSON,
+            jobType ENUM('page_text', 'extraction') NOT NULL,
+            createdAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+        )'''
+    ))
+    conn.commit() 
+    index = conn.execute(text('''
+        SELECT COUNT(*)
+        FROM information_schema.statistics
+        WHERE TABLE_SCHEMA = DATABASE()
+        AND TABLE_NAME = 'jobs' 
+        AND INDEX_NAME = 'idx_userid_createdat'; 
+    '''))
+    if int(index.first()[0]) == 0:
+        print('creating index: ')
+        conn.execute(text('CREATE INDEX idx_userid_createdat ON jobs(user_id, createdAt)'))
```

Comment:
This made me LOL. is annoying that there is no create index if not exists in mysql